draft-ietf-scitt-architecture.md

```diff
@@ -179,19 +179,19 @@ Consumer of Signed Statements:
 Envelope:
 
 : metadata and an Issuer's signature is added to a Statement via a COSE Envelope by the Issuer to produce a Signed Statement.
-An Envelope contains the identity of the Issuer and other information to help components responsible for validation that are part of a Transparency Services to identify the software Artifact referred to in a Signed Statement.
+The Envelope contains the identity of the Issuer and information about the Artifact to help Transparency Services to properly validate and process the Signed Statement.
 In essence, a Signed Statement is a COSE Envelope wrapped around a Statement binding the metadata included in the Envelope to a Statement.
 In COSE, an Envelope consists of a protected header (included in the Issuer's signature) and an unprotected header (not included in the Issuer's signature).
 
 Feed:
 
-: an identifier chosen by the Issuer for the Artifact.
-For every Issuer and Feed, the Registry on a Transparency Service contains a sequence of Signed Statements about the same Artifact.
+: a logical collection of Statements about the same Artifact.
+For any step or set of steps in a supply chain there will be multiple statements made about the same Artifact. Issuers use the Feed to create a coherent sequence of Signed Statements about the same Artifact and Verifiers use the Feed to ensure completeness and non-equivocation in supply chain evidence by identifying all Transparent Statements linked to the one(s) they are evaluating.
```

Comment:
I checked the definition of Verifier as it starts to move into more focus now. Pulling the definition in here for convenience:
```
an entity that consumes Transparent Statements (a specialization of Signed Statement Consumer),
verifying their proofs and inspecting their Statement payload, either before using corresponding
Artifacts, or later to audit an Artifact's provenance on the supply chain.
```

With respect to the quoted definition:
1.) `specialization of Signed Statement Consumer` seems to be outdated?
2.) what kind of proofs in `verifying their proofs`?
3.) payload opaque to the TS is inspected in `inspecting their Statement payload`?
4.) what does `before using corresponding Artifacts` mean?

With respect to the proposed changes in line 189:
1.) how do Verifiers `ensure completeness`? Is that via `verifying their proofs`?
2.) `Statements linked to the one(s) they are evaluating` is done via the Feed value, right?

Reply:
> Quoted definition on verifier:

@henkbirkholz, should this be a separate PR to add any [clarification for Verifier within the terminology](https://github.com/ietf-wg-scitt/draft-ietf-scitt-architecture/blob/main/draft-ietf-scitt-architecture.md?plain=1#L244)? 

> 1.) how do Verifiers `ensure completeness`? Is that via `verifying their proofs`?
> 2.) `Statements linked to the one(s) they are evaluating` is done via the Feed value, right?

My interpretation of the following: 
> Verifiers use the Feed to ensure completeness and non-equivocation in supply chain evidence by identifying all Transparent Statements linked to the one(s) they are evaluating.

A consumer|verifier can query the statements about the ~feed~ subject. They may choose to query all statements, filter the statements by specific issuers, filter by specific contentTypes, or a mix of the each. But, all are related to a specific artifact, as defined by a ~feed~ subject.  Because they've identified "the artifact" by a subject

Reply:
@henkbirkholz when we say related to the same artifact, does this imply a historical record/tracking type of report for the "same artifact"? Thanks.

Reply:
@rjb4standards,
As an example:
Wabbit-networks ships the `net-monitor v1` software package.
They wish to provide:
- SPDX SBOM
- ClycloneDX SBOM
- A VEX report
- Updated VEX reports as they learn more
- A statement when the product will no longer be maintained for support
- A statement that says it's being replaced by v1.2

All those are statements that Wabbit-networks may make on that specific artifact.

Other entities, such as Cosmic Security or MITRE can also make statements about the same artifact.

As a consumer, you would use the ~Feed~ | Subject to correlate all the statements about the same Artifact The `net-monitor v1` package.

Reply:
The stitching of coherent information from multiple sources is complex.  The view from within a corporation could be the public published information augmented with the internal Legal or security endorsements.  Thus the thought that all relevant content flows through one Transparency services does not work here.  In addition, the decision of whether one identity can make statements about another's product is not under the control of the Transparency service.   The feed is not going to be authoritative in this context.

Reply:
> Thus the thought that all relevant content flows through one Transparency services does not work here. 

Agreed. This is also why we need a property for multiple instances of SCITT to have a property to correlate upon :) 

Resolving this, and open to additional Issues/PRs for additional clarity.